### PR TITLE
Native bridge

### PR DESCRIPTION
**Pull request by ben-clayton**

### CODE REVIEW

File: core/os/android/adb/device.go [hunk 1]
```diff
@@ -254,3 +254,53 @@ func parseDevices(ctx context.Context, out string) (map[string]bind.Status, erro
 	}
 	return devices, nil
 }
+
+// NativeBridgeABI returns the native ABI for the given emulated ABI for the
+// device by consulting the ro.dalvik.vm.isa.<emulated_isa>=<native_isa>
+// system properties.
```
Comment (pmuetschard) [inline core/os/android/adb/device.go]: I think this should also say that it returns the passed in ABI if there is no emulation.

Comment (pmuetschard) [reply core/os/android/adb/device.go]: never mind... for some reason GH was showing me out of date information and this PR was still open.

Comment (ben-clayton) [reply core/os/android/adb/device.go]: Still a good point. Will fix shortly.

Comment (ben-clayton) [reply core/os/android/adb/device.go]: https://github.com/google/gapid/pull/1016/files

**Pull request merged**